openghg/store/_flux.py

```diff
@@ -378,3 +406,38 @@ def validate_data(data: Dataset) -> None:
         """
         data_schema = Flux.schema()
         data_schema.validate_data(data)
+
+    def chunking_schema(
+        self,
+        time_resolved: bool = False,
+        high_time_resolution: bool = False,
+    ) -> ChunkingSchema:
+        """
+        Get chunking schema for footprint data.
+
+        Args:
+            time_resolved : Set footprint variable to be high time resolution.
+            high_time_resolution: This argument is deprecated and will be replaced in future versions with time_resolved.
+
+        Returns:
+            dict: Chunking schema for footprint data.
+        """
+
+        if high_time_resolution:
```

Comment:
Question over whether we need to include the `high_time_resolution` / `time_resolved` input here?

For `Flux` this concept this concept doesn't make as much sense as it does for `Footprint` so moving away from this labelling where we can would be good.